includes/admin/reporting/class-export-payments.php

```diff
@@ -130,8 +130,8 @@ public function get_data() {
 					$id = isset( $payment_meta['cart_details'] ) ? $download['id'] : $download;
 
 					// If the download has variable prices, override the default price
-					$price_override = isset( $payment_meta['cart_details'] ) ? $download['price'] : null;
-
+					$price_override = isset( $payment_meta['cart_details'] ) ? $download['subtotal'] : null;
```

Comment:
This will need an isset() check on the `subtotal`. Old payment records didn't have that key.